src/components/scoresheets/scoresheet.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { removeScoresheet } from '../../actions/scoresheet_actions';
+
+import './scoresheet.css';
+
+//=========================================
+// props / actions
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    scoresheet: ownProps.scoresheet,
```

Comment:
If scoresheet is already coming down via classic React props (ownProps) there's no need to use Redux to map it to props. Let's talk about this, make sure you are clear on React / Redux division which is the most common area of confusion for recent grads anyway!